File: Practice problem/return middle word/return middle word.py
# 연습문제 가운데 글자 가져오기 Level 1

def solution(s):
    if len(s) % 2 == 1:
	    return s[(len(s)) // 2]
    else: 
        return s[(len(s)-1) // 2:((len(s)-1) // 2)+2]

s = solution("abcde")
print(s)
s = solution("wear")
print(s)

File: Practice problem/return middle word/return middle word.cpp
// 연습문제 가운데 글자 가져오기 Level 1

#include <string>

using namespace std;

string solution(string s) {
	if (s.size() % 2 == 1) { return s.substr(s.size() / 2, 1); }
	else { return s.substr((s.size() - 1) / 2, 2); }
}
